plugins/org.jboss.tools.livereload.ui/src/org/jboss/tools/livereload/ui/internal/command/OpenInWebBrowserViaLiveReloadProxyEnablementTester.java

```diff
@@ -34,7 +35,11 @@ public boolean test(Object receiver, String property, Object[] args, Object expe
 			return false;
 		}
 		final IServer appServer = appModule.getServer();
-		return appServer != null && WSTUtils.isServerStarted(appServer);
+		IModule[] modules = appModule.getModule();
+		return appServer != null
+				&& appServer.getModuleState(appModule.getModule()) == IServer.STATE_STARTED
```

Comment:
@fbricon, see [this JIRA](https://issues.jboss.org/browse/JBIDE-21936). It was said to disable for stopped modules. So i've made a check that module is started. Do we really need to make it enabled for unknown state?